target/arduino-due.hpp

```diff
@@ -18,6 +19,20 @@ namespace hwstl {
     using pin_sequence = std::integer_sequence<pin_index, vt_pins...>;
 
     namespace arduino_due {
+#ifdef DEBUG
```

Comment:
I would prefer using `DEBUG_ENABLE` instead of `DEBUG`, as `DEBUG_ENABLE` would give a better understanding why this preprocessor global is used.